staff/templates/staff_base.html

```diff
@@ -93,6 +94,15 @@
       });
     });
     </script>
+    <script>
+      window.onload = function(){
```

Comment:
This code is duplicated, should be factored out into a common template or at least a common script include.

Reply:
In "manipulation_js.html"